manifests/init.pp

```diff
@@ -119,6 +119,14 @@
     order   => '01',
   }
 
+  if $includes {
+    concat::fragment { 'dhcp.conf+50_includes':
+      target  => "${dhcp_dir}/dhcpd.conf",
+      content => template('dhcp/dhcpd.conf.includes.erb'),
+      order   => '01',
```

Comment:
I think this might need to be somewhere from 11-49, so it's ordered immediately after the failover dhcpd.conf file, but before the dhcp_class define (as it used to be).